docs/contributing/style-guide.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Style Guide
+parent: Contributing
+nav_order: 4
+---
+
+# Style Guide
+
+This is a general guide to the code style of our commits. This is by no means an exhaustive list and we're constantly
+changing and improving it. This is also a work in progress document.
+
+For our commit messages please see our [Commit Message Guidelines](./commitmsg-guidelines.md).
+
+## Tools
+
+We implement the following tools that help us abide by our style guide and include the configuration for them inside
+our repository:
+- [golangci-lint](https://github.com/golangci/golangci-lint)
```

Comment:
We should also add eslint in here as that enforces style for all `js` `ts` and `tsx` files along with prettier.